src/components/withLocalize.js

```diff
@@ -140,6 +158,19 @@ class LocaleContextProvider extends React.Component {
         return LocaleDigitUtils.fromLocaleDigit(this.props.preferredLocale, localeDigit);
     }
 
+    /**
+     * @param {String} currencyCode
+     * @return {String}
+     */
+    toLocalizedCurrencySymbol(currencyCode) {
+        const parts = NumberFormatUtils.formatToParts(this.props.preferredLocale, 0, {
+            style: 'currency',
+            currency: currencyCode,
+        });
+        const currencySymbol = _.find(parts, part => part.type === 'currency').value;
+        return currencySymbol;
+    }
+
```

Comment:
Let's add unit tests for this method. 
1. Run the test for each currency in the ONYXKEYS.CURRENCY_LIST. The test fails when it does not return a value.

Reply:
Currency list is fetched from server. How do you suggest creating unit test? I did manual test everything works as expected.

Reply:
Currency list is fetched from server. How do you suggest creating unit test? I did manual test everything works as expected.

Reply:
Download that list. Create a JSON object and then use it in the Unit test.

Reply:
Yeah, I would do the same that @parasharrajat but maybe only do it with a small portion of the list. Just with a couple of currencies that test the different cases.

Reply:
The end motive is that every currency should pass the test otherwise App will crash on runtime. I guess the list is small enough to fit in JSON so we should add full.

Reply:
Yeah, that is the case. But very likely. At least we will have something to prevent.